sources/packages/backend/workflow/test/2021-2022/parttime-assessment/costs/parttime-assessment-2021-2022-additional-transportation-costs.e2e-spec.ts

```diff
@@ -0,0 +1,370 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import { OfferingDeliveryOptions } from "@sims/test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-additional-transportation-costs.`, () => {
+  it(
+    "Should determine calculatedDataTotalTransportationAllowance when studentDataAdditionalTransportKm is zero " +
+      "and offeringDelivered is onsite",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataAdditionalTransportKm = 0;
+      assessmentConsolidatedData.offeringWeeks = 30;
+      assessmentConsolidatedData.offeringDelivered =
+        OfferingDeliveryOptions.Onsite;
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+      // Assert
+      // calculatedDataTotalTransportationAllowance = 13 * 30 = 390
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataTotalTransportationAllowance,
+      ).toBe(390);
+    },
+  );
+
+  it(
+    "Should determine calculatedDataTotalTransportationAllowance when studentDataAdditionalTransportKm is zero " +
+      "and offeringDelivered is online",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataAdditionalTransportKm = 0;
+      assessmentConsolidatedData.offeringDelivered =
+        OfferingDeliveryOptions.Online;
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+      // Assert
+      expect(
+        calculatedAssessment.variables
+          .calculatedDataTotalTransportationAllowance,
+      ).toBe(0);
+    },
+  );
+
+  it(
+    "Should determine calculatedDataTotalTransportationAllowance when studentDataAdditionalTransportKm less than 280 " +
```

Comment:
less than 280 and greater than 1 km?

Reply:
It is updated to use "less than 280 and greater than 1km" in my latest commit. Please have a look.